theatre/studio/src/uiComponents/Popover/usePopover.tsx

```diff
@@ -88,6 +93,9 @@ export default function usePopover(
               threshold: opts.pointerDistanceThreshold ?? 100,
               callback: () => {
                 if (lock.childHasFocusRef.current) return
+                // this is a bit weird, because when you stop capturing, then the popover can close on you...
+                // TODO: Better fixes?
```

Comment:
Only way to fix this corner case imo would be to use pointer lock instead of pointer capture. Locking would keep the pointer in place, not just eliminating this corner case, but entirely eliminating the need for this logic. But locking can be buggy (see number input) so probably not for now.